### PR TITLE
Joinus whitespace fix

### PR DESCRIPTION
**Pull request by ericliddle**

### CODE REVIEW

**Review by wtLau — CHANGES_REQUESTED** [review]
need to fix the margin space between header and content to reflect consistency

**Comment by wtLau** [comment]
footer changes is also in this pull-request

File: themes/sgsc/sass/_footer.scss [hunk 1]
```diff
@@ -141,6 +141,10 @@ span.wpcf7-not-valid-tip {
 }
 
 // Footer
+.site-footer {
+  border-top: 1px dashed lightgray;
```
Comment (kmsolberg) [inline themes/sgsc/sass/_footer.scss:145]: Can you remove the border from the desktop & tablet size? It's only necessary for mobile width.

**Review by kmsolberg — CHANGES_REQUESTED** [review]

**Comment by kmsolberg** [comment]
I agree with narrowing the margin space, it needs to be consistent with the other pages - the design has no margin between the image and the header 
image

File: themes/sgsc/sass/_footer.scss (hunk 1 above)
Comment (ericliddle) [reply themes/sgsc/sass/_footer.scss:145]: Removed footer above 690px when footer changes from mob.

**Comment by ericliddle** [comment]
@kmsolberg The styling on your page completely off.  The links above the title are different now, and my local and staging site show appropriate spacing: 

image

**Comment by kmsolberg** [comment]
Yes the menu doesn't look like that anymore but the images should still touch the orange header

**Comment by ericliddle** [comment]
@kmsolberg done and done

**Review by wtLau — APPROVED** [review]

**Review by kmsolberg — APPROVED** [review]

**Pull request merged**